public/js/gogs.js

```diff
@@ -830,6 +830,10 @@ $(document).ready(function () {
 
     // Highlight JS
     if (typeof hljs != 'undefined') {
+        // creating aliases for languages
```

Comment:
Maybe you can use of this? https://github.com/gogits/gogs/blob/master/modules/template/highlight.go